merlin/data/dataorganization.py

```diff
@@ -31,40 +33,56 @@ class DataOrganization(object):
     image files.
     """
 
-    def __init__(self, dataSet, filePath: str = None):
+    def __init__(self, dataSet, filePath: str = None,
+                 dataPortal: dataportal.DataPortal = None):
```

Comment:
Should there be a way to add a dataportal when constructing an analysis folder with merlin.py?